addon/routes/index.js

```diff
@@ -12,6 +12,7 @@ export default class IndexRoute extends Route {
     search: PARAM_OPTIONS,
     document: PARAM_OPTIONS,
     activeGroup: PARAM_OPTIONS,
+    listView: PARAM_OPTIONS,
```

Comment:
When switching from list to grid view, do you really need a model refresh? The data is the same, isn't it?

Reply:
updated